crypto/cipher/aes256cbc/conversions_test.go

```diff
@@ -64,7 +64,7 @@ func Test_asPrivateECIES(t *testing.T) {
 			"success-secp256k1-pointer",
 			args{
 				func() *secp256k1.PrivateKey {
-					k, err := secp256k1.PrivateKeyFromHex("01901E63389EF02EAA7C5782E08B40D98FAEF835F28BD144EECF5614A415943F")
+					k, err := secp256k1.PrivateKeyFromBytes("01901E63389EF02EAA7C5782E08B40D98FAEF835F28BD144EECF5614A415943F")
```

Comment:
`secp256k1.PrivateKeyFromBytes` is expecting `[]byte` You need to first do a `hex.DecodeString("01901E63389EF02EAA7C5782E08B40D98FAEF835F28BD144EECF5614A415943F")` check the error then pass it to  `secp256k1.PrivateKeyFromBytes`.